src/native/fs/fs_napi.cpp

```diff
@@ -186,41 +233,92 @@ struct Stat : public FSWorker
     }
     virtual void OnOK()
     {
-        DBG1("FS::Stat::OnOK: " << DVAL(_path) << DVAL(_stat_res.st_ino) << DVAL(_stat_res.st_size));
+        DBG1("FS::Stat::OnOK: " << DVAL(_path) << DVAL(_stat_res.st_ino) << DVAL(_stat_res.st_size));    
+        Napi::Env env = Env();
+        auto res = set_stat_res(env, _stat_res);
```

Comment:
The function name is misleading. 
If you want to `set_stat_res()`, then you should send in the object on which to set.
Currently the function is creating and populating the object, so a more suitable name `is make_stat_res()`
Personally I would prefer that the callers will create the object and pass it in to set_stat_res() and then you don't need to return it (no radical auto in function response type...)